mlir/include/mlir/Dialect/LLVMIR/LLVMEnums.td

```diff
@@ -705,4 +705,56 @@ def FramePointerKindEnum : LLVM_EnumAttr<
   let cppNamespace = "::mlir::LLVM::framePointerKind";
 }
 
+//===----------------------------------------------------------------------===//
+// RoundingMode
+//===----------------------------------------------------------------------===//
+
+def RoundTowardZero
```

Comment:
I assume this matches some pre-existing list in LLVM. Would it make sense to link that here?